docs/developer-guide/tests/sqlness-test.md

```diff
@@ -38,4 +38,13 @@ cargo run --bin sqlness-runner
 ```
 It automatically finishes the following procedures: compile `GreptimeDB`, start it, grab tests and feed it to
 the server, then collect and compare the results. You only need to check whether there are new `.output` files.
-If not, congratulations, the test is passed 🥳!
\ No newline at end of file
+If not, congratulations, the test is passed 🥳!
+
+### Run a specific test
+
+```shell
+cargo run --bin sqlness-runner your_test
+```
+
+If you specify a second argument, only test cases containing the specified string in their names will be executed.
```

Comment:
and sqlness also support filter on env. Filter is accepted as regex string. And case name will be examine in form of `env:case`.

Reply:
@waynexia, do we need to include this statement in the doc?

Reply:
Yes, please. I rephrase it to:

sqlness also supports filtering based on environment. The filter is accepted as a regex string and the case name will be examined in the format of 'env:case'.